astropy/coordinates/tests/accuracy/generate_spectralcoord_ref.py

```diff
@@ -39,29 +40,33 @@
 
     for row in tab:
         # Produce input file for rv command
-        with open("rv.input", "w") as f:
-            f.write(
-                f"{row['obslon'].to_string('deg', sep=' ')}"
-                f" {row['obslat'].to_string('deg', sep=' ')}\n"
+        rv_input = Path("rv.input")
+        rv_input.write_text(
+            "".join(
```

Comment:
do these need a join?
String joining happens automatically if the strings are on consecutive lines.

Reply:
thanks. Indeed there was room for simplification here